proto/opamp.proto

```diff
@@ -635,7 +649,10 @@ enum AgentCapabilities {
     AgentCapabilities_ReportsHealth                  = 0x00000800;
     // The Agent will report RemoteConfig status via AgentToServer.remote_config_status field.
     AgentCapabilities_ReportsRemoteConfig            = 0x00001000;
-
+    // The Agent can report heartbeats on a default interval of 30s.
```

Comment:
```suggestion
    // The Agent will report heartbeats on a default interval of 30s.
```
I think "can" is not strong enough. It implies a possibility, but the server needs an assurance.